src/utils.js

```diff
@@ -23,4 +23,26 @@ export function elementWidth(el) {
  */
 export function lockBodyScrolls(lock) {
   document.body.style.overflow = lock ? 'hidden' : '';
-}
\ No newline at end of file
+}
+
+/**
+ * takes a number and returns it as a human readable number
+ * exp (10000 = 10k)
```

Comment:
Example is `e.g. ` 
```suggestion
 * e.g. (10000 = 10k) 
```

Reply:
I would more examples of their weird rules because now the function looks needlessly complicated. Like

```js
1200 => 1,200
12000 => 12k
```

Reply:
Yea, ill add them now